### ISSUE TITLE
Question/Bug: List’s `listItemHeight` not working? (iOS)

### ISSUE THREAD

**Issue by oferRounds** [GeekyAnts/NativeBase#1398]
react-native, react and native-base version: 2.3.3

Expected behaviour: item’s height should change according to List’s `listItemHeight` property

Actual behaviour: item’s height always stays the same

Steps to reproduce: regular static list

Tested on iOS

**Comment by SupriyaKalghatgi** [comment]
Fixed with NativeBase [2.3.5](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.5)

**Comment by oferRounds** [comment]
YAY!!! Thanks!

**Comment by oferRounds** [comment]
Wait, you just removed it? So it means it can’t be controlled?

**Comment by SupriyaKalghatgi** [comment]
Yes, usage of this was discontinued, though it was not removed from theme

**Comment by oferRounds** [comment]
Ok. Then how can I increase the height of the list item?

**Comment by akhil-geekyants** [comment]
@oferRounds you can just override the style of ListItem and set the height to your desired height.
Like  `<ListItem style={{ height: 60 }}>`

**Comment by oferRounds** [comment]
Ok, tried it, but now there is an issue with the separator. Any idea how to solve it?

<img width="429" alt="screen shot 2017-12-11 at 8 50 27" src="https://user-images.githubusercontent.com/1477963/33818973-937059e4-de50-11e7-9c19-7023febd0c38.png">

**Comment by oferRounds** [comment]
```
render() {
    return (
      <Container>
        <Content>
          <List listItemHeight = { 60 }>
            <Separator bordered>
              <Text>HELP</Text>
            </Separator>
            <ListItem style = {{ height: kCellHeight }} icon onPress = { () => Linking.openURL(Consts.HELP_CENTRE_URL) }>
              <Body>
                <Text>Help Centre</Text>
              </Body>
              <Right>
                <Icon name = 'arrow-forward' />
              </Right>
            </ListItem>
            <ListItem
              icon
              style = {{ height: kCellHeight }}
              onPress = { () => Linking.openURL('mailto:contact@hibob.com?subject=Please tell us how we can help!')}>
              <Body>
                <Text>Contact Bob Team</Text>
              </Body>
              <Right>
                <Icon name = 'arrow-forward' />
              </Right>
            </ListItem>
            <Separator bordered>
              <Text>LEGAL</Text>
            </Separator>
            <ListItem icon style = {{ height: kCellHeight }} onPress = { () => Linking.openURL(Consts.PRIVACY_POLICY_URL) }>
              <Body>
                <Text>Privacy Policy</Text>
              </Body>
              <Right>
                <Icon name = 'arrow-forward' />
              </Right>
            </ListItem>
            <ListItem icon style = {{ height: kCellHeight }} onPress = { () => Linking.openURL(Consts.TERMS_OF_SERVICE_URL)}>
              <Body>
                <Text>Terms of Service</Text>
              </Body>
              <Right>
                <Icon name = 'arrow-forward' />
              </Right>
            </ListItem>
          </List>
        </Content>
        <Footer>
          <FooterTab>
            <Button onPress = { this._handleLogOutButtonTap.bind(this) }><Text style = {{ color: '#E33054' }}>Log Out</Text></Button>
          </FooterTab>
        </Footer>
      </Container>
    )
  }
```

**Comment by akhil-geekyants** [comment]
@oferRounds same way as we did for <ListItem/>. Just override the style of <Separator/> and set the desired height.
Like  `<Separator bordered style={{ height: 50 }}>`

**Comment by oferRounds** [comment]
Thanks, but I get a tslint error, saying that `style` does not exist on this item. Also, it doesn’t seem to affect even I insist on adding it

**Comment by oferRounds** [comment]
Oh, sorry, I see that it does affect, but it just increases the seperator; my problem is with the right side of the list item

**Comment by oferRounds** [comment]
Ok, solved! Adding it to both `Body` and `Right` items solved it. I think it’s much much better to add a support as intended before, have one place where you control the item’s height (as with the native iOS)

**Comment by oferRounds** [comment]
Also, as mentioned before, if want to change the `Separator`’s height, I get a tslint error. How can this be solved?

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.3.6](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.6)
